provider/aws/processes.go

```diff
@@ -1191,10 +1192,38 @@ func (p *Provider) generateTaskDefinition2(app, service string, opts structs.Pro
 		}
 	}
 
+	td, err := p.stackResource(fmt.Sprintf("%s-%s", p.Rack, app), "ServiceWeb")
```

Comment:
our cluster doesn't have service `web` so it throw error `ERROR: resource not found: ServiceWeb` when we do `convox run [other service name]`

Reply:
@numcys can you fix this

Reply:
@khoinguyenm we have released the fix: https://github.com/convox/rack/releases/tag/20241104181809
Thanks for your patience